gril/grilunsol.c

```diff
@@ -39,6 +39,32 @@
 #include "common.h"
 #include "grilunsol.h"
 
+int g_ril_unsol_parse_connected(GRil *gril, const struct ril_msg *message)
+{
+	struct parcel rilp;
+	int size;
+	int version = RIL_VERSION_UNSPECIFIED;
+
+	DBG("");
```

Comment:
My vote would be no, don't change the rest of them.

If we want to cleanup DBG() statements in order to make the code more consistent, then let's do so in a dedicated pull-request.  I don't like combining cleanup and functional changes in the same pull-request.   Also before submitting such a pull-request, this is probably worth some discussion as to how we want to clean things up, before spending the time creating a branch...